hicommonwealth/commonwealth
# #5506 - Create upvote popover

This PR completes ticket #5506, which asks for the creation of a new upvote popover component. The upvote popover contains a list of users that have upvoted a thread. It also details the total number of voters and the voting weight of each individual voter.

## Link to Issue
Closes: #5506 

## Description of Changes
- creates component `UpvotePopover`
- adds `UpvotePopover` to components showcase

## Test Plan
- navigate to `/components`
- go to 'Upvote Popover' section and check that behavior of component is as expected
